test/test_gesv.cc

```diff
@@ -221,8 +221,9 @@ void test_gesv_work(Params& params, bool run)
 
     slate::Pivots pivots;
 
-    slate::generate_matrix(params.matrix,  A);
-    slate::generate_matrix(params.matrixB, B);
+    slate::Options matgen_opts = {{slate::Option::Target, target}};
+    slate::generate_matrix(params.matrix,  A, matgen_opts);
+    slate::generate_matrix(params.matrixB, B, matgen_opts);
```

Comment:
Why not just use the `opts` used for `getrf`, etc.?

Reply:
I was trying to get identical matrices, and `panel_threads` and `ib` both modify the order of operations (and thus round off).  Although, `geqrf` is already slightly non-deterministic for identical options, so it's probably a moot point.

I can switch it to `opts` if you think that's better.

Reply:
Makes sense to keep identical matrices.